specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -1023,6 +1070,9 @@ application component types can be clients of RMI objects.
 
 ===== RMI-IIOP (Proposed Optional)
 
+*TODO Note:* Since distributed interop is being removed from the EJB spec, do we remove
```

Comment:
I would tentatively say, "yes", but I would need to review the EJB spec's usage of RMI-IIOP.

I know there are specific requirements for non-CORBA protocols that say you must not conflict with RMI-IIOP and I believe the term was "RMI-IIOP compatible."  We would want to keep those types of usages because we do have implementations that can only support remote EJBs via CORBA.

Reply:
Will wait with deleting this section until we complete this discussion on the mailing list.